client/src/components/Register.vue

```diff
@@ -14,7 +14,7 @@
             <br><br>
             <button type="submit" value="Submit">Register</button>
             <div>
-                Register Message: {{ registerStatus }}
+            Register Message: {{ registerStatus }}
```

Comment:
This should be written like <div>sometexthere</div>
All on one line, not split.

Reply:
`<div>sometexthere</div>`
All on one line, github ate my markup.